language/move-prover/tests/sources/stdlib/transaction_scripts/add_validator.exp

```diff
@@ -0,0 +1,8 @@
+Move prover returns: exiting with checking errors
+error:
+
+   ┌── tests/sources/stdlib/transaction_scripts/add_validator.move:4:3 ───
+   │
+ 4 │   0x0::LibraSystem::add_validator(new_validator);
```

Comment:
If we include the scripts, we would need to add the `// dep: path` to let them resolve imports. But on the other hand, the prover does nothing right now with scripts, so perhaps we shouldn't copy them now, but at the time we add support for scripts. (We should open an issue to add script support.)

Reply:
I removed the scripts.